helloworld-jest-test-sample/src/tests/runTest.ts

```diff
@@ -0,0 +1,37 @@
+import * as path from 'path';
+import { runTests } from 'vscode-test';
+
+async function main() {
+	try {
+		// The folder containing the Extension Manifest package.json
+		// Passed to `--extensionDevelopmentPath`
+		const extensionDevelopmentPath = path.resolve(__dirname, '../../');
+
+		// The path to the extension test script
+		// Passed to --extensionTestsPath
+		const extensionTestsPath = path.resolve(
+			__dirname,
+			'../../node_modules/vscode-jest-test-runner'
+		);
+
+		// The path to a module that runs some code to configure or set up the
+		// testing framework before each test.
+		// See https://github.com/bmealhouse/vscode-jest-test-runner#jest_test_runner_setup
+		// process.env.JEST_TEST_RUNNER_SETUP = path.resolve(
+		// 	__dirname,
+		// 	'./custom-vscode-jest-test-runner-setup.js'
+		// );
```

Comment:
Would you mind pointing either to VS Code website in this section or moving https://github.com/bmealhouse/vscode-jest-test-runner#jest_test_runner_setup over to the README and point to it?

We would like to make sure all links link to official Microsoft resources.

Reply:
NVM. I just realized that you published your own test-runner, and it makes sense to have documentation there.
Would you be interested in merging the code to vscode-test repo?

Reply:
Yes, I will look into moving this code over to the vscode-test repo and updating the docs accordingly.

Reply:
BTW, you can use `extensionTestsEnv`, see https://github.com/microsoft/vscode-test

Reply:
I've opened up a draft PR to move the runner over to `vscode-test`: https://github.com/microsoft/vscode-test/pull/59

After I complete testing, I will mark it as ready for review.

Reply:
I have completed my testing. After merging and releasing https://github.com/microsoft/vscode-test/pull/59, the only thing remaining for this PR is to upgrading dependencies.